css/style.css

```diff
@@ -0,0 +1,6 @@
+#image {
+  max-width: 100%;
+  height: auto;
+  display: block;
+  margin:auto;
+}
```

Comment:
Hi @stevenmukama 

Good job so far!

**Changes required**

1. Install and use linters 
2. Use classes to add styles in css